shopping_mall/backend/app/models/faq_doc.py

```
@@ -0,0 +1,129 @@
+            prefix = f"[{self.faq_category.name}] "
+        return f"{prefix}Q: {self.title}\nA: {self.content}"
+
+    def to_chroma_metadata(self) -> dict:
+        """ChromaDB document metadata를 생성합니다."""
+        import json as _json
+
+        meta: dict = {}
+        try:
+            meta = _json.loads(self.extra_metadata or "{}")
+        except Exception:
+            pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)